src/features/Tiles/MovieTiles/index.js

```diff
@@ -44,7 +45,7 @@ const MovieTiles = ({ title, popularMoviesData, movieData }) => {
                                 )}
 
                                 <Tags>
-                                    {genres.map(tag => <Tag key={`genres-${tag.name}`}>{tag.name}</Tag>)}
+                                    {moviesList[index].genre_ids.map(movie => <Tag key={`genres-${movie}`}>{moviesGenres.find(({ id }) => id === movie).name}</Tag>)}
```

Comment:
This line is too long on my screen so maybe needs some formatting.

I'm also a bit unsure about the names you use here.
`moviesList[index]`: I don't think you need index here. Just call it movie like in the code above.
`movie`: It's not really a movie but genreID
There are many ways to write it but I would do something like that:
`{movie.genre_ids.map(genreID => <Tag key={`genres-${genreID}`}>{moviesGenres.find(({ id }) => id === genreID).name}</Tag>)}`
or maybe `{genreID}` is enough in key...

Reply:
true. I`ve forgotten that its already in map and we can use "movie" to get inside.